integration-tests/test_207.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
can you run this file in CI too (and other integration tests that missed) please?

Reply:
I added this tests. There are no other tests that we can run automatically right now - the remaining tests require user input. They'll need a bit of work before they can be fully automatic.

Reply:
Aha, ok, we can do it later (out of scope of current PR)